randovania/games/samus_returns/logic_database/Area 1.txt

```diff
@@ -146,24 +149,27 @@ Extra - asset_id: collision_camera_005
   * Extra - actor_type: doorchargecharge
   > Door from Water Maze
       Trivial
-  > Dock to Harmonized Hallway
+  > Door to Harmonized Hallway
       Trivial
   > Dock to Chute Leech Cabin
       Trivial
-  > Dock to Water Maze
+  > Door to Water Maze
       Trivial
 
 > Door from Water Maze; Heals? False
   * Layers: default
-  * Access Closed to Water Maze/Door to Gullugg Gangway
+  * Access Closed to Water Maze/Door to Gullugg Gangway (Bottom)
```

Comment:
For reference: It needs a migration because of this renamed node because it will break the rdvgames of next month stable.
I think that's also the only one.